classes/note/NoteDAO.php

```diff
@@ -81,13 +86,24 @@ public function getByUserId($userId, $rangeInfo = null)
      * @param int $orderBy Optional sorting field constant: self::NOTE_ORDER_...
      * @param int $sortDirection Optional sorting order constant: SORT_DIRECTION_...
      *
-     * @return object DAOResultFactory containing matching Note objects
+     * @return LazyCollection<Note>
      */
-    public function getByAssoc($assocType, $assocId, $userId = null, $orderBy = self::NOTE_ORDER_DATE_CREATED, $sortDirection = self::SORT_DIRECTION_DESC, $isAdmin = false)
+    public function getByAssoc(int $assocType, int $assocId, ?int $userId = null, bool $isAdmin = false, 
```

Comment:
Could you change this back to the original parameter ordering? That way it won't confuse callers who don't expect it to change (e.g. in third party plugins), and it'll match the parameter order in the self-documentation.

Reply:
Sure, changed